### DISCUSSION TITLE
Thonny on Mac use Raspberry Pi to connect to ESP32

### DISCUSSION THREAD

**Discussion by ghost** [thonny/thonny#3239] [General]
Hello. How do I do this: I want to code on my MacBook but I do not want to connect any boards to my it, so I use my Raspberry Pi 3 for that. However even though the Pi has Thonny on remote desktop I would like to keep my development on my Mac. So how do I connect the Thonny on my Mac to the MCU ESP32 on my Pi? 
I have RealVNC running on Pi and ssh works. 

<img width="1366" alt="IMG_2600" src="https://github.com/thonny/thonny/assets/69945037/0d13d01a-77e7-4d84-8863-8ebab40a35d5">

**Comment by aivarannamaa** [comment]
You could try make Thonny SSH to your RPi -- see Tools => Options => Interpreter => Remote Python 3
